drivers/telescope/lx200ap_experimental.cpp

```diff
@@ -544,15 +610,52 @@ bool LX200AstroPhysicsExperimental::ISNewSwitch(const char *dev, const char *nam
     if (!strcmp(name, UnparkFromSP.name))
     {
         IUUpdateSwitch(&UnparkFromSP, states, names, n);
-        int unparkPos = IUFindOnSwitchIndex(&UnparkFromSP);
-
-        LOGF_DEBUG("Unpark from pos set to (%d).", unparkPos);
+        ParkPosition unparkPos = (ParkPosition)IUFindOnSwitchIndex(&UnparkFromSP);
 
-        UnparkFromSP.s = IPS_OK;
+	UnparkFromSP.s = IPS_OK;
+        if( unparkPos != PARK_LAST)
+        {
+            double unparkAlt, unparkAz;
+            if (!calcParkPosition(unparkPos, &unparkAlt, &unparkAz))
+            {
+                LOG_DEBUG("ISNewSwitch: Error calculating unpark position!");
+                UnparkFromSP.s = IPS_ALERT;
+            }
+            LOGF_DEBUG("ISNewSwitch: parkPos=%d parkAlt=%f parkAz=%f", unparkPos, unparkAlt, unparkAz);
+            SetAxis1ParkDefault(unparkAz);
+            SetAxis2ParkDefault(unparkAlt);
+        }
+        // 2020-06-01, wildi, UnPark() relies on it
+        saveConfig(true);
         IDSetSwitch(&UnparkFromSP, nullptr);
         return true;
     }
 
+    // ===========================================================
+    // Switch Park(ed), Unpark(ed)
+    // ===========================================================
+    // 2020-06-02, wildi, ToDo, discuss with Jasem
+    if (!strcmp(name, ParkSP.name))
+    {
+        IUUpdateSwitch(&ParkSP, states, names, n);
+        int sw = IUFindOnSwitchIndex(&ParkSP);
+        if(sw == 0)
+        {
+            Park();
+            IUResetSwitch(&ParkSP);
+            ParkS[0].s = ISS_ON;
+        }
+        else
+        {
+            UnPark();
+            IUResetSwitch(&ParkSP);
+            ParkS[1].s = ISS_ON;
+        }
+
+        ParkSP.s = IPS_OK;
+        IDSetSwitch(&ParkSP, nullptr);
+        return true;
```

Comment:
Why override INDI::Telescope handling? because of the IPS_ALERT?

Reply:
Because I do not like the idea that the mount is unparked on startup. This mode is ok, if there is somebody beside the mount. In unattended case it could be dangerous.

E.g. if the mount is physically at park 2 position and INDI thinks it is at park 3 and Last Park is set then the set co-ordinates are wrong. 

Further I noticed that using INDI::Telescope Parking is set to IPS_OK, Park(ed) and if I click on UnPark(ed) it says:

2020-06-06T08:13:41: [INFO] Telescope already unparked. 
but the success message:

LOGF_INFO("UnPark: Current parking position ...

did not appear.

As I wrote in the source, that's a matter to be discussed.